atlasdb-client/src/main/java/com/palantir/atlasdb/schema/TargetedSweepSchema.java

```diff
@@ -137,6 +137,25 @@ private static void addSweepStateTables(Schema schema) {
                 conflictHandler(ConflictHandler.IGNORE_ALL);
             }
         });
+
+        schema.addTableDefinition("sweepAssignedBuckets", new TableDefinition() {
+            {
+                javaTableName("SweepAssignedBuckets");
+                allSafeForLoggingByDefault();
+                rowName();
+                hashFirstNRowComponents(3);
+                rowComponent("shard", ValueType.VAR_LONG);
+                rowComponent("major_bucket_identifier", ValueType.VAR_LONG);
```

Comment:
It might be more consistent to call it coarse and fine? I worry that, since the major vs coarse and minor vs fine numbers are so far apart in size that they'll cause confusion.

Reply:
It _is_ more consistent, but I would argue here that we _don't_ want consistency as they actually refer to different concepts!